firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -34,14 +34,16 @@ public expect val Firebase.firestore: FirebaseFirestore
 /** Returns the [FirebaseFirestore] instance of a given [FirebaseApp]. */
 public expect fun Firebase.firestore(app: FirebaseApp): FirebaseFirestore
 
-public expect class NativeFirebaseFirestore
+internal expect class NativeFirebaseFirestore
 
 public class FirebaseFirestore internal constructor(private val wrapper: NativeFirebaseFirestoreWrapper) {
 
-    public constructor(native: NativeFirebaseFirestore) : this(NativeFirebaseFirestoreWrapper(native))
+    public companion object {}
+
+    internal constructor(native: NativeFirebaseFirestore) : this(NativeFirebaseFirestoreWrapper(native))
 
     // Important to leave this as a get property since on JS it is initialized lazily
-    public val native: NativeFirebaseFirestore get() = wrapper.native
+    internal val native: NativeFirebaseFirestore get() = wrapper.native
```

Comment:
This used to be a common level alternative to `android/`ios`/`js` but it is now internal, because `NativeFirebaseFirestore` is now internal